doc/source/tune/_tutorials/tune-sklearn.py

```diff
@@ -0,0 +1,147 @@
+from sklearn import datasets
+import numpy as np
+
+# Load in data
+digits = datasets.load_digits()
+
+# Set training and test sets
+x = digits.data
+y = digits.target
+x_train, x_test, y_train, y_test = train_test_split(x, y, test_size=.2)
+
+# Example parameters to tune from SGDClassifier
+parameter_grid = {"alpha": [1e-4, 1e-1, 1], "epsilon": [0.01, 0.1]}
+
+#######################################################################
+# As you can see, the setup here is exactly how you would do it for Scikit-Learn. Now, let's try fitting a model.
+
+tune_search = TuneGridSearchCV(
+    SGDClassifier(),
+    parameter_grid,
+    scheduler="MedianStoppingRule",
+    early_stopping=True,
+    max_iters=10)
+
+import time  # Just to compare fit times
+start = time.time()
+tune_search.fit(x_train, y_train)
+end = time.time()
+print("Tune Fit Time:", end - start)
+
+#######################################################################
+# Note the slight differences we introduced above:
+#
+#  * a `scheduler`, and
+#  * a specification of `max_iters` parameter
+#
+# The ``scheduler`` determines when to stop early - MedianStoppingRule is a great default, but see :ref:`Tune's documentation on schedulers <tune-schedulers>` here for a full list to choose from. ``max_iters`` is the maximum number of iterations a given hyperparameter set could run for; it may run for fewer iterations if it is early stopped.
+#
+# Try running this compared to the GridSearchCV equivalent, and see the speedup for yourself!
+
+from sklearn.model_selection import GridSearchCV
+# n_jobs=-1 enables use of all cores like Tune does
+sklearn_search = GridSearchCV(SGDClassifier(), parameter_grid, n_jobs=-1)
+
+start = time.time()
+sklearn_search.fit(x_train, y_train)
+end = time.time()
+print("Sklearn Fit Time:", end - start)
+
+###################################################################
+# Using Bayesian Optimization
+# ---------------------------
+#
+# In addition to the grid search interface, tune-sklearn also provides an interface, TuneSearchCV, for sampling from **distributions of hyperparameters**.
+#
+# In addition, you can easily enable Bayesian optimization over the distributions in only 2 lines of code:
+
+from tune_sklearn.tune_search import TuneSearchCV
+from sklearn.linear_model import SGDClassifier
+from sklearn import datasets
+from sklearn.model_selection import train_test_split
+from ray.tune.schedulers import MedianStoppingRule
+import numpy as np
+
+digits = datasets.load_digits()
+x = digits.data
+y = digits.target
+x_train, x_test, y_train, y_test = train_test_split(x, y, test_size=.2)
+
+clf = SGDClassifier()
+parameter_grid = {"alpha": (1e-4, 1), "epsilon": (0.01, 0.1)}
+
+scheduler = MedianStoppingRule(grace_period=10.0)
+
+tune_search = TuneSearchCV(
+    clf,
+    parameter_grid,
+    search_optimization="bayesian",
+    n_iter=3,
+    scheduler=scheduler,
+    max_iters=10,
+)
+tune_search.fit(x_train, y_train)
```

Comment:
Should we have some output here too? It can finish running this but no output comes out of it

Reply:
what do you mean by this? what should be changed?

Reply:
Oh just some print statement after fitting to show some results. It can fit properly but the script will finish fitting with no printing. Should we add something like that?

Reply:
yeah printing sounds good, any suggestions?

Reply:
Perhaps just `print(tune_search.cv_results_)` is good

Reply:
```suggestion
tune_search.fit(x_train, y_train)
print(tune_search.cv_results_)
```